src/IJ_mv/IJMatrix_parcsr.c

```diff
@@ -2984,7 +2984,7 @@ hypre_IJMatrixSetValuesOMPParCSR( hypre_IJMatrix       *matrix,
 
    HYPRE_Int print_level = hypre_IJMatrixPrintLevel(matrix);
    //HYPRE_Int max_num_threads;
-   HYPRE_Int error_flag = 0;
+   //HYPRE_Int error_flag = 0;
```

Comment:
We probably should get rid of this and replace it with the proper hypre_error messages.

Reply:
The reason I wasn't sure what to do here is because, at the end of the routine, ther is an `if (error_flag)` then `return hypre_error_flag` that was commented out (I assume for a good reason).  So, should that `return` be replaced with a `hypre_error_w_msg` or something (and uncommented)?  Any advice here is welcome.

Reply:
@ulrikeyang @victorapm : Any suggestions about what to do with this file, or should we just leave it as is?  Thanks again!

Reply:
We cannot just comment this out, because we are adding on this variable, and so this would break the code. We would have to remove all the error_flag++ occurrences. The best thing would be to clean it up using hypre_error stuff. Right now there are also print statements depending on the print_level.

Reply:
This doesn't break the code.  `error_flag` is never used in this routine.  It's set and incremented, and that's it.  There is a block of code at the bottom of the routine that does use `error_flag`, but it's commented out (not by me).  These changes are guaranteed safe and won't change the behavior of the code.

I don't know how to clean up the error stuff here, because the commented-out block near the bottom was apparently commented out for a reason, and I don't understand what that reason was.  From my visual inspection, I would say we should keep the `error_flag` stuff and uncomment the `if (error_flag)` thing at the bottom, but that may cause issues with regression tests and users.

Reply:
It doesn't look commented out to me. Are we looking at different files? What about line 3097 etc?

Reply:
3097 is commented out:
https://github.com/hypre-space/hypre/blob/cccdab8543c9972c2d697b10700812f02e78dae8/src/IJ_mv/IJMatrix_parcsr.c#L3097

The `if` statement I mention is at the bottom of the routine here:
https://github.com/hypre-space/hypre/blob/cccdab8543c9972c2d697b10700812f02e78dae8/src/IJ_mv/IJMatrix_parcsr.c#L3558-L3569